src/moderatorFrontendMain/kotlin/util/Strings.kt

```diff
@@ -7,600 +7,610 @@ import com.studo.campusqr.common.utils.LocalizedString
 object Strings {
 
   val edit = LocalizedString(
-    "Edit",
-    "Bearbeiten"
+      "Edit",
+      "Bearbeiten"
   )
 
   val delete = LocalizedString(
-    "Delete",
-    "Löschen"
+      "Delete",
+      "Löschen"
   )
 
   val copy = LocalizedString(
-    "Copy",
-    "Kopieren"
+      "Copy",
+      "Kopieren"
   )
 
   val duplicate = LocalizedString(
-    "Duplicate",
-    "Duplizieren"
+      "Duplicate",
+      "Duplizieren"
   )
 
   val locations_element_download_qr_code = LocalizedString(
-    "Print / download QR code",
-    "QR-Code drucken / herunterladen"
+      "Print / download QR code",
+      "QR-Code drucken / herunterladen"
   )
 
   val locations_element_simulate_scan = LocalizedString(
-    "Simulate QR code scan",
-    "Scan des QR-Codes simulieren"
+      "Simulate QR code scan",
+      "Scan des QR-Codes simulieren"
   )
 
   val locations_element_download_csv = LocalizedString(
-    "Download check ins as Excel .csv file",
-    "Check-Ins als Excel .csv Datei herunterladen"
+      "Download check ins as Excel .csv file",
+      "Check-Ins als Excel .csv Datei herunterladen"
   )
 
   val error_try_again = LocalizedString(
-    "Something went wrong. Please try again.",
-    "Etwas lief schief. Bitte versuchen Sie es nochmal."
+      "Something went wrong. Please try again.",
+      "Etwas lief schief. Bitte versuchen Sie es nochmal."
   )
 
   val locations = LocalizedString(
-    "Locations",
-    "Orte"
+      "Locations",
+      "Orte"
   )
 
   val location_delete = LocalizedString(
-    "Delete location",
-    "Ort löschen"
+      "Delete location",
+      "Ort löschen"
+  )
+
+  val location_delete_are_you_sure = LocalizedString(
+      "Are you sure you want to delete this location? This will also delete all the Access controls, linked with this location.",
+      "Sind Sie sicher, dass Sie diesen Ort löschen möchten? Dadurch werden auch alle Zugangskontrollen gelöscht, die mit diesem Standort verknüpft sind."
```

Comment:
```suggestion
      "Are you sure you want to delete this location? This will also delete all the access permissions linked with this location.",
      "Sind Sie sicher, dass Sie diesen Ort löschen möchten? Dadurch werden auch alle Zugangsberechtigungen gelöscht, die mit diesem Ort verknüpft sind."
```